.github/workflows/e2e.yaml

```diff
@@ -74,7 +69,38 @@ jobs:
           distribution: "temurin"
           java-version: ${{ matrix.java }}
           cache: "maven"
-      - name: Run ${{ matrix.broker }} Platform ${{ matrix.kafka }} E2E Tests
+      - name: Run Confluent Kafka Platform ${{ matrix.kafka }} E2E Tests
+        run: |
+          echo "Running integration tests with Docker Compose file: ${COMPOSE_FILE}"
+          docker compose -f ${COMPOSE_FILE} pull
+          mvn --batch-mode --no-transfer-progress --errors --update-snapshots clean verify -Drevision=$(git describe --tags --always)
+
+  e2e-redpanda:
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        java: [ "11", "17" ]
+        kafka: ["3.7.1","3.8.0"]
+        redpanda: ["v24.1.8"]
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 15
+    env:
+      AWS_ACCESS_KEY_ID: test
+      AWS_SECRET_ACCESS_KEY: test
+      AWS_DEFAULT_REGION: us-east-1
```

Comment:
nit: can we move those to global env variables because they're the same across all jobs, right? See https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/variables#defining-environment-variables-for-a-single-workflow